legal-api/src/legal_api/models/filing.py

```diff
@@ -717,16 +721,18 @@ def get_incomplete_filings_by_types(business_id: int, filing_types: list):
         return filings
 
     @staticmethod
-    def get_a_businesses_most_recent_filing_of_a_type(business_id: int, filing_type: str):
+    def get_a_businesses_most_recent_filing_of_a_type(business_id: int, filing_type: str, filing_sub_type: str = None):
         """Return the filings of a particular type."""
         max_filing = db.session.query(db.func.max(Filing._filing_date).label('last_filing_date')).\
             filter(Filing._filing_type == filing_type). \
+            filter(Filing._filing_sub_type == filing_sub_type). \
```

Comment:
This will always include `filing_sub_type` in query (`filing_sub_type = None` or `filing_sub_type = 'value'`). It will restrict me to get a most recent restoration filing without `filing_sub_type`. Is it intended?

Reply:
Not intended.  Will fix.